README.md

```diff
@@ -537,6 +538,19 @@ abstract class AbstractEnumType extends Type
 
 Override those methods in order to satisfy your needs.
 
+## Symfony HttpKernel component
+<a href="https://symfony.com"><img src="https://img.shields.io/badge/Symfony-3.1%2B-green.svg?style=flat-square" title="Available for Symfony 3.1+" alt="Available for Symfony 3.1+" align="right"></a>
+
+There is an [argument value resolver](https://symfony.com/doc/current/controller/argument_value_resolver.html) provided,
+which seamlessly transforms HTTP request parameter into enum instance, just by typehinting the target enum in controller action.
+
+Simply register it inside the DIC configuration:
+```yaml
+services:
+    Elao\Enum\Bridge\Symfony\HttpKernel\Controller\ArgumentResolver\EnumValueResolver:
+        tags: [{ name: controller.argument_value_resolver }]
```

Comment:
Shouldn't the priority be set higher than 100 in order to be executed before `RequestAttributeValueResolver`, so you can still get a request attribute value as enum instead of raw value?